marketplace_docs/templates/Fabric/scripts/01-initial-setup

```diff
@@ -35,3 +35,9 @@ rm -f /etc/zulip/zulip-secrets.conf /etc/zulip/settings.py
 #
 # For more details see https://chat.zulip.org/#narrow/stream/3-backend/topic/apt-daily
 systemctl disable apt-daily.timer apt-daily-upgrade.timer
+
+# Configure firewall. Else, img_check.sh will warn that ufw is not configufred.
+ufw allow ssh
+ufw allow http
+ufw allow https
```

Comment:
Hmm, do we want to allow the incoming email port?  I believe that's needed for the incoming email gateway.

Reply:
Added port 25.

Reply:
Can you add a comment by the port 25 one noting it's for the incoming email gateway, with a link to the docs?  And then feel free to merge.

Reply:
I have added the link to the docs. Don't have merge rights for this so one of you folks has to merge.